### ISSUE TITLE
Sphinx documentation looks different locally and on readthedocs.io

### ISSUE THREAD

**Issue by david-berthelot** [google/objax#72]
The documentation on https://objax.readthedocs.io/en/latest/ doesn't match what we get when building in `docs` results. It seems some options are ignored, notably whether to put type hints in the signature or in the arguments description.

For example: https://objax.readthedocs.io/en/latest/objax/nn.html#objax.nn.BatchNorm
Compared the function signature and parameters section with one locally built.
